index.html

```diff
@@ -569,7 +579,11 @@ <h3>Extensions to the HTMLElement interface</h3>
                         </dl>
                         <ol>
                             <li>If [=this=]'s [=Element/local name=] is neither a [=valid shadow host name=] nor "<code>canvas</code>", then [=exception/throw=] a {{"NotSupportedError"}} {{DOMException}}.</li>
+                            <li>If |editContext|'s <a>associated element</a> is equal to [=this=], then terminate these steps.
+                            <li>If |editContext|'s <a>associated element</a> is not null, then [=exception/throw=] a {{"NotSupportedError"}} {{DOMException}}.</li>
```

Comment:
just to make sure, this implies that if authors want to change an EditContext's associated element, they will need to first call oldElement.editContext=null, then newElement.editContext=editContext, correct? do we want to do it for the authors?

Reply:
Yes, because this makes it forward-compatible if we want to change things later so that an EditContext can be associated with multiple elements. If we handle the switch for the authors automatically, then code could break if we later change the behavior to add a second element to the EditContext.

But if `newElement.editContext = editContext` throws today, then it's easier for us to make a change later so that `newElement.editContext = editContext` succeeds and adds the second element.

That's my thinking on it anyway.

Reply:
makes sense to me, thanks for the explanation.